tests/table.spec.ts

```diff
@@ -139,6 +139,176 @@ test.describe('tests complex table operations', () => {
 
     // Check snapshot
     await expect(page.locator('.iris-grid-column')).toHaveScreenshot();
+
+    await page
+      .locator('.iris-grid')
+      .locator('data-testid=btn-page-close')
+      .first()
+      .click();
+
+    await expect(page.locator('.iris-grid .table-sidebar')).toHaveCount(0);
+  });
+
+  test('can conditional format', async ({ page }) => {
+    await page.goto('');
+    const consoleInput = page.locator('.console-input');
+    await consoleInput.click();
+
+    const command = `${makeTableCommand(
+      undefined,
+      TableTypes.StringAndNumber
+    )}`;
+
+    await pasteInMonaco(consoleInput, command);
+    await page.keyboard.press('Enter');
+
+    // Wait for the panel to show
+    await expect(page.locator('.iris-grid-panel')).toHaveCount(1);
+
+    // Wait until it's done loading
+    await expect(page.locator('.iris-grid-panel .loading-spinner')).toHaveCount(
+      0
+    );
+
+    // Model is loaded, need to make sure table data is also loaded
+    await expect(
+      page.locator('.iris-grid .iris-grid-loading-status')
+    ).toHaveCount(0);
+
+    const tableOperationsMenu = page.locator(
+      'data-testid=btn-iris-grid-settings-button-table'
+    );
+    await tableOperationsMenu.click();
+
+    // Wait for Table Options menu to show
+    await expect(page.locator('.table-sidebar')).toHaveCount(1);
+
+    // Open Conditional Formatting Panel
+    await page.locator('data-testid=menu-item-Conditional Formatting').click();
+
+    // Create New Formatting Rule
+    await page.getByRole('button', { name: 'Add New Rule' }).click();
+    await page.getByPlaceholder('Enter value').click();
+    await page.keyboard.type('3');
+    await page.getByRole('button', { name: 'No formatting' }).click();
```

Comment:
Wonder if there's a better way to select this... right now it's looking for a button that says "No formatting", when really the instructions we want are "click the dropdown labelled 'Style'". Perhaps we can update our accessibility labels/roles to make it more intuitive.